src/features/DashboardPage/index.jsx

```diff
@@ -40,6 +40,28 @@ const DashboardPage = () => {
     fetchExams();
   }, []); // eslint-disable-line react-hooks/exhaustive-deps
 
+  const handleCancelExam = async (vueAppointmentId) => {
```

Comment:
This function has the same structure as handleGetScoreReportUrl, there is a way to avoid code duplication?

Reply:
Yes, I noticed that too, I was thinking of opening a new PR to fix it once the two changes were merged so I could have a better understanding of where each change goes, what do you think?

Reply:
I added an approach in my pr, if you are ok with that, we have to merge first that and just add your functions with the params